spec/models/target_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe Target, type: :model do
+  subject { create(:target) }
+
+  describe 'validations' do
+    it { should validate_presence_of(:title) }
+    it { should validate_presence_of(:radius) }
+    it { should validate_presence_of(:latitude) }
+    it { should validate_presence_of(:longitude) }
+    it { should validate_numericality_of(:radius).is_less_than_or_equal_to(1000) }
+    it { should validate_numericality_of(:radius).is_greater_than(0) }
```

Comment:
I think you can do 
`validate_numericality_of(:radius).is_less_than_or_equal_to(1000).is_greater_than(0)`

Reply:
Great! Thank you i tried it and that works perfectly!